experimental/dds/tree/src/test/utilities/TestUtilities.ts

```diff
@@ -283,10 +285,17 @@ async function setUpLocalServerTestSharedTreeGeneric<
 
 	const treeId = id ?? 'test';
 	const registry: ChannelFactoryRegistry = [[treeId, factoryGetter(summarizeHistory)]];
+	const innerRequestHandler = async (request: IRequest, runtime: IContainerRuntimeBase) =>
```

Comment:
General question across the board: Any reason `rootDataStoreRequestHandler` won't work instead of `innerRequestHandler ` ?